.buildkite/pipelines/pull-request/bwc-snapshots-windows.yml

```diff
@@ -15,6 +15,6 @@ steps:
         agents:
           provider: gcp
           image: family/elasticsearch-windows-2022
-          machineType: custom-32-98304
+          machineType: n1-standard-32
```

Comment:
Hmm, are we hitting OOM on Windows? We don't build in ram on those machines so I wouldn't expect us to.